pkg/rust/cargo/parse.go

```diff
@@ -9,13 +12,14 @@ import (
 	"golang.org/x/xerrors"
 )
 
+type cargoPkg struct {
+	Name         string   `toml:"name"`
+	Version      string   `toml:"version"`
+	Source       string   `toml:"source,omitempty"`
+	Dependencies []string `toml:"dependencies,omitempty"`
+}
 type Lockfile struct {
-	Packages []struct {
-		Name         string   `toml:"name"`
-		Version      string   `toml:"version"`
-		Source       string   `toml:"source,omitempty"`
-		Dependencies []string `toml:"dependencies,omitempty"`
-	} `toml:"package"`
+	Packages []cargoPkg `toml:"package"`
 	Metadata interface{}
```

Comment:
If we still use `toml` decoder:

we don't use tag for `Metadata`, but toml decoder still parses this field:
![image](https://user-images.githubusercontent.com/91113035/225511867-74f4f83a-f679-4e2e-8038-8660c985d91a.png)
Maybe it make sense to remove `Metadata`?